src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.targets

```diff
@@ -2270,16 +2270,9 @@ because xbuild doesn't support framework reference assemblies.
 
   <Copy SourceFiles="%(ApkFiles.FullPath)" DestinationFolder="$(OutDir)" />
 
-  <GetFilesThatExist
-      Condition="'$(AndroidManagedSymbols)' == 'True' "
-      Files="@(_ResolvedAssemblies->'%(Identity)');@(_ResolvedAssemblies->'%(Identity).mdb');@(_ResolvedAssemblies->'%(Identity).msym')">
-    <Output TaskParameter="FilesThatExist" ItemName="_SymbolicateArchiveAssemblies" />
-  </GetFilesThatExist>
-
-  <Copy Condition=" '$(AndroidManagedSymbols)' == 'True' And Exists('%(_SymbolicateArchiveAssemblies.Identity)') "
-    SourceFiles="@(_SymbolicateArchiveAssemblies)"
-    DestinationFolder="$(OutDir)$(_AndroidPackage).apk.msym"
-    SkipUnchangedFiles="true"
+  <Exec
+    Command="mono-symbolicate store-symbols $(OutDir)$(_AndroidPackage).apk.msym $(MonoAndroidIntermediate)android/assets"
```

Comment:
Filesystem paths should be quoted, if (when) they contain spaces:

    Command="mono-symbolicate store-symbols &quot;$(OutDir)$(_AndroidPackage).apk.msym&quot; &quot;$(MonoAndroidIntermediate)android\assets&quot;"

Additionally, this should probably specify the directory containing `mono-symbolicate`.